src/components/ReportWelcomeText.tsx

```diff
@@ -1,92 +1,65 @@
 import * as ReportUtils from '@libs/ReportUtils';
-import reportPropTypes from '@pages/reportPropTypes';
 import CONST from '@src/CONST';
 import ONYXKEYS from '@src/ONYXKEYS';
 import ROUTES from '@src/ROUTES';
+import type {PersonalDetailsList, Policy, Report} from '@src/types/onyx';
 import Text from './Text';
 import UserDetailsTooltip from './UserDetailsTooltip';
-import withLocalize, {withLocalizePropTypes} from './withLocalize';
-
-const personalDetailsPropTypes = PropTypes.shape({
-    /** The login of the person (either email or phone number) */
-    login: PropTypes.string,
-
-    /** The URL of the person's avatar (there should already be a default avatar if
-  the person doesn't have their own avatar uploaded yet, except for anon users) */
-    avatar: PropTypes.string,
-
-    /** This is either the user's full name, or their login if full name is an empty string */
-    displayName: PropTypes.string,
-});
-
-const propTypes = {
-    /** The report currently being looked at */
-    report: reportPropTypes,
-
-    /** The policy object for the current route */
-    policy: PropTypes.shape({
-        /** The name of the policy */
-        name: PropTypes.string,
-
-        /** The URL for the policy avatar */
-        avatar: PropTypes.string,
-    }),
-
-    /* Onyx Props */
 
+type ReportWelcomeTextOnyxProps = {
     /** All of the personal details for everyone */
-    personalDetails: PropTypes.objectOf(personalDetailsPropTypes),
-
-    ...withLocalizePropTypes,
+    personalDetails: OnyxEntry<PersonalDetailsList>;
 };
 
-const defaultProps = {
-    report: {},
-    policy: {},
-    personalDetails: {},
+type ReportWelcomeTextProps = ReportWelcomeTextOnyxProps & {
+    /** The report currently being looked at */
+    report: Report;
+
+    /** The policy for the current route */
+    policy: Policy;
 };
 
-function ReportWelcomeText(props) {
+function ReportWelcomeText({report = {} as Report, policy = {} as Policy, personalDetails = {}}: ReportWelcomeTextProps) {
+    const {translate} = useLocalize();
     const styles = useThemeStyles();
-    const isPolicyExpenseChat = ReportUtils.isPolicyExpenseChat(props.report);
-    const isChatRoom = ReportUtils.isChatRoom(props.report);
+    const isPolicyExpenseChat = ReportUtils.isPolicyExpenseChat(report);
+    const isChatRoom = ReportUtils.isChatRoom(report);
     const isDefault = !(isChatRoom || isPolicyExpenseChat);
-    const participantAccountIDs = lodashGet(props.report, 'participantAccountIDs', []);
+    const participantAccountIDs = report?.participantAccountIDs ?? [];
     const isMultipleParticipant = participantAccountIDs.length > 1;
     const displayNamesWithTooltips = ReportUtils.getDisplayNamesWithTooltips(
-        OptionsListUtils.getPersonalDetailsForAccountIDs(participantAccountIDs, props.personalDetails),
+        // TODO: Remove type assertion (`as PersonalDetailsList`) after `src/libs/OptionsListUtils.js` is migrated into ts
```

Comment:
The comment is not aligned with the [TS Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/TS_STYLE.md#migration-guidelines).

Reply:
@fabioh8010 I tried to find the issue for `TS migration of OptionsListUtils`, but was not able to find it, so commented `TODO` like above.
Could you kindly point out the issue number for it?
OR will it be okay to remove this `TODO` comment here?

Reply:
Thanks, addressed by [e9e32fb](https://github.com/Expensify/App/pull/33251/commits/e9e32fb838fd88414827deac4fb9690e8bdd0a3f)